File: fine_tuning/results/test_orig_files/question-8-pass-1.py
def getMax(arr, N, K):

def getMax(arr, N, K):
    if K == 0:
        return arr[0]
    if N == 1:
        return max(arr)
    dp = [0] * (N + 1)
    dp[1] = arr[0]
    for i in range(2, N + 1):
        dp[i] = max(dp[i - 1], dp[i - 2] + arr[i - 1])
    return dp[N]

File: feedback_pipeline/generations/cpp/files_with_feedback/question-8-pass-1.cpp
#include <bits/stdc++.h>
using namespace std;
int getMax(const int arr[], int N, int K) {
    int max_val = arr[0], max_idx = 0;
    for (int i = 1; i < N; i++) {
        if (arr[i] > max_val) {
            max_val = arr[i];
            max_idx = i;
        }
    }
    return max_val;
}